cornflow-server/cornflow/tests/unit/test_executions.py

```diff
@@ -115,7 +115,9 @@ def test_new_execution_no_instance(self):
         self.assertTrue("error" in response.json)
 
     def test_get_executions(self):
-        self.get_rows(self.url, self.payloads)
+        keys_to_check = ['instance_id', 'schema', 'indicators', 'id', 'config', 'data_hash', 'basic_log_json',
+                         'created_at', 'state', 'user_id', 'message', 'name', 'description']
+        self.get_rows(self.url, self.payloads, keys_to_check=keys_to_check)
```

Comment:
Should it make sense to at least add this keys to check argument to all the test where the payload is not checked?